daos/authentication.js

```diff
@@ -0,0 +1,7 @@
+const mongoose = require('mongoose');
```

Comment:
I'm not going to request changes on these, but we don't actually need any of these files yet. We just need the empty app. The DAOs should be added when that issue is being added.

Reply:
Sorry, I got a little ahead of myself there it seems.